app/ui/src/app/support/support.component.ts

```diff
@@ -0,0 +1,158 @@
+import { Component,  Input,  ViewChild, ElementRef, OnInit } from '@angular/core';
+
+import { ObjectPropertyFilterConfig } from '../common/object-property-filter.pipe';
+
+import { Restangular } from 'ngx-restangular';
+import { Http } from '@angular/http'
+
+import { Integrations, Integration } from '../integration/integration.model';
+import { IntegrationStore } from '../store/integration/integration.store';
+import { IntegrationSupportService } from '../integration/integration-support.service';
+import { log, getCategory } from '../logging';
+import { Observable } from 'rxjs/Observable';
+
+import fileSaver = require("file-saver");
+
+const SUPPORT_FORM_CONFIG = {
+  platformLogs: {
+    displayName: 'Platform Logs',
+    type: 'boolean',
+    description: 'Include logs of platform services',
+    default: false
+  },
+  integrationLogs: {
+    displayName: 'Integrations Logs',
+    type: 'boolean',
+    description: 'Include logs of integrations',
+  }
+};
+
+@Component({
+  selector: 'syndesis-support',
+  templateUrl: './support.component.html',
+  styleUrls: ['./support.component.scss']
+})
+export class SupportComponent implements OnInit {
+
+  // Used to bind patternfly behavior with foreign form elements
+  @ViewChild('alllogs') allogs: ElementRef;
+  @ViewChild('specificlogs') specificlogs: ElementRef;
+
+  allLogsSelected : boolean = true;
+  loading : boolean= true;
+
+  private restangularService: Restangular;
+
+  filter: ObjectPropertyFilterConfig = {
+    filter: '',
+    propertyName: 'name'
+  };
+
+  // List configuration
+  listConfig = {
+    multiSelect: true,
+    selectItems: false,
+    showCheckbox: true
+  };
+
+  // Toolbar configuration
+  toolbarConfig = {
+    filterConfig: {
+      fields: [
+        {
+          id: 'name',
+          title: 'Name',
+          placeholder: 'Filter by Name...',
+          type: 'text'
+        },
+        {
+          id: 'description',
+          title: 'Description',
+          placeholder: 'Filter by Description...',
+          type: 'text'
+        }
+      ]
+    }
+  };
+
+  items: Array<any> = [
+    {
+        name: "Integration Name 1",
+        description: "Description about the integration goes here"
+    },
+    {
+        name: "Integration Name 2",
+        description: "Description about the integration goes here"
+    }
+  ];
+
+  constructor(
+    public store: IntegrationStore,
+    public integrationSupportService: IntegrationSupportService,
+  ) {
+  }
+  
+  buildData(data: any = {}) {
+    console.log(data);
+    this.integrationSupportService
+      .downloadSupportData(data)
+      .subscribe(response => {
+        fileSaver.saveAs(response, "syndesis.zip");
+      },
+      error => {
+        console.log("Error downloading the file.");
+        console.log(error);
+      }
+    );
+    return {  };
+  }
+
+  // Handles events when the user interacts with the toolbar filter
+  filterChanged($event) {
```

Comment:
`$event` is the reserved template keyword for event handlers, but we usually do rename it into `event` in the Angular component class, since there some early drafts for future ECMAScript implementation that might turn into using special char-prefixed names for native system features.

In other words, I'd rename this to plain `event` instead (you have to leave `$event` in the template though).